cmd/network/packet-capture.go

```diff
@@ -325,3 +363,138 @@ func copyFilesFromPacketCapturePods(o *packetCaptureOptions) error {
 
 	return nil
 }
+
+// desiredPacketCapturePod returns the desired Pod read in from manifests
+func desiredPacketCapturePod(o *packetCaptureOptions, key types.NamespacedName) *corev1.Pod {
+	capturePod := &corev1.Pod{}
+	t := true
```

Comment:
Because the 	SecurityContext: &corev1.SecurityContext{Privileged: &t} in line 404. takes a pointer. I cannot pass the value true directly. this is why the extra variable was needed :'(

Reply:
Ooof, sorry, I missed that.